### PR TITLE
Syncing algorithms with liboqs.

### PR DESCRIPTION
**Pull request by xvzcf**

### CODE REVIEW

File: oqs_template/generate.yml [hunk 1]
```diff
@@ -176,290 +133,98 @@ sigs:
...
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256192ssimple'
+    family: 'SPHINCS-SHA2'
+    name: 'sphincssha2192ssimple'
     oid: '1 3 9999 6 5 7'
     oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x05, 0x07'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_192s_simple'
-    code_point: '0xfe70'
+    oqs_meth: 'OQS_SIG_alg_sphincs_sha2_192s_simple'
+    code_point: '0xfebb'
```
Comment (baentsch) [inline oqs_template/generate.yml:165]: Code point is in sync with oqsprovider, but OID is not: Intentional? Irrelevant? Is this getting interop-tested somehow (against what?)?

Comment (xvzcf) [reply oqs_template/generate.yml:165]: I fixed the OIDs, and as far as I know, we haven't done interop testing in a while have we? I can add it back to BoringSSL if need be.

Comment (baentsch) [reply oqs_template/generate.yml:165]: > add it back to BoringSSL if need be.

I'd say it only needs to be if we want to keep supporting BoringSSL and Chromium. Do we? @dstebila ?@christianpaquin?

Comment (christianpaquin) [reply oqs_template/generate.yml:165]: If I remember our last discussion correctly: we can support it on a best-effort basis; i.e., fix things when we notice them breaking (vs. full-fledge interop test). I did some interop testing recently between boringssl and openssl when I last updated the project in Dec 2022; but mainly using Kyber. We already provide a strong disclaimer in the README, we can add more details about the state of our interop testing.

**Review by baentsch — COMMENTED** [review]
Worthwhile also adding reference to oqsprovider? Just to make sure no-one expects interop there, either.

**Review by dstebila — APPROVED** [review]

**Review by baentsch — APPROVED** [review]

**Pull request merged**